src/methods/get.js

```diff
@@ -128,11 +129,14 @@ export function OrdinaryGet(
       // descValue unless it is empty.
       // Only get the parent value if it does not involve a getter call.
       // Use a property get for the joined value since it does the check for empty.
+      let [hash, args] = hashBinary("!==", descValue, realm.intrinsics.empty);
```

Comment:
Observation: This block of code builds up a binary condition around `!==`. This is a particular instantiation of the more general binary expression construction. And the string "!==" is mentioned three times here...